CHANGELOG.md

```diff
@@ -6,9 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+* `UrlResolver#resolve()` returns urls containing search and hash
```

Comment:
"search" -> "query string", "hash" -> "fragment"

Reply:
Updated in CHANGELOG and in the mocha test names.

Reply:
I had just used the property from node's Url object.  I'll use querystring and fragment happily.